script/DeployOracle.s.sol

```diff
@@ -116,3 +113,57 @@ contract DeployWstethRethL2ValidityOracles is Script, CommonMainnet {
     vm.stopBroadcast();
   }
 }
+
+// BROADCAST
+// source .env && forge script DeployCamelotOdUsdOracle --with-gas-price 2000000000 -vvvvv --rpc-url $ARB_MAINNET_RPC --broadcast --verify --etherscan-api-key $ARB_ETHERSCAN_API_KEY --sender $DEFAULT_KEY_PUBLIC_ADDRESS --account defaultKey
+
+// SIMULATE
+// source .env && forge script DeployCamelotOdUsdOracle --with-gas-price 2000000000 -vvvvv --rpc-url $ARB_MAINNET_RPC --sender $DEFAULT_KEY_PUBLIC_ADDRESS
+
+contract DeployCamelotOdUsdOracle is Script, CommonMainnet {
+  IBaseOracle public _odEthCamelotRelayer;
```

Comment:
public var shouldn't have an underscore. just function scoped or internal vars should have underscore